hashcompute.go

```diff
@@ -43,7 +43,7 @@ func AverageHash(img image.Image) (*ImageHash, error) {
 // http://www.hackerfactor.com/blog/?/archives/529-Kind-of-Like-That.html
 func DifferenceHash(img image.Image) (*ImageHash, error) {
 	if img == nil {
-		return nil, errors.New("Image object can not be nil")
+		return nil, errors.New("image object can not be nil")
```

Comment:
nit: Is there any reason to change this?

Reply:
golang code review recommendations:

Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
This is just a small detail, my code linter picks these up.